articles/language/statements.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Q# statements | Microsoft Docs 
-description: Q# statements
+title: Q# Statements
+description: Learn about the correct use of statements in Q# code, including comments, namespaces, statement blocks and others. 
```

Comment:
I'd suggest that function/operation declarations, variable declarations and assignments, and operation calls are all more important to list here than namespaces?